src/PropertySuggester/Suggesters/SimpleSuggester.php

```diff
@@ -169,7 +170,15 @@ public function suggestByItem( Item $item, $limit, $minProbability, $context ) {
 					);
 				}
 
-				$numericEntityId = $dataValue->getEntityId()->getNumericId();
+				$entityId = $dataValue->getEntityId();
+
+				if ( !$entityId instanceof Int32EntityId ) {
```

Comment:
One of the few cases where I would put brackets to make order or evaluation more clear